lab_1_keywords_tfidf/main.py

```diff
@@ -33,7 +44,14 @@ def remove_stop_words(tokens: list[str], stop_words: list[str]) -> Optional[list
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if isinstance(tokens, list) or isinstance(stop_words, list):
```

Comment:
what if tokens is empty? 
what if it contains values of the wrong type?

Reply:
it will return None?

Reply:
it should, yes
but as of now, it isn't stated cleary